lib/src/models/credit_card_config.dart

```diff
@@ -2,6 +2,7 @@
 class CreditCardConfig {
   ///  An option to save (tokenize) the card after a successful payment.
   bool saveCard;
+  bool manual;
```

Comment:
Can you please add a one-liner doc comment just like the `saveCard` variable above?

Reply:
sure 👍🏻